### ISSUE TITLE
Support for CatBoost?

### ISSUE THREAD

**Issue by lextoumbourou** [TeamHG-Memex/eli5#276]
Hi there,

I'd love to see support for [CatBoost](https://github.com/catboost/catboost) in eli5. It has a Scikit-style API like XGBoost and LightGBM, so should be a lot of code we can copy and paste from those implementations.

I might have a dig into it myself, if no one else is already looking at it.

❤️

**Comment by AshwinB-hat** [comment]
Hey,
I would like to take up this issue if no one is on it!
It will be my first issue fyi.

**Comment by lopuhin** [comment]
Nice! Adding ``explain_weights`` support would be a good start here 👍

**Comment by AshwinB-hat** [comment]
Yes,
Sounds like a good milestone to start with!

**Comment by AshwinB-hat** [comment]
I have raised a PR for `explain_weights' #303

**Comment by ivanprado** [comment]
@AshwinB-hat added support for `explain_weights` (Thanks! :-) ) but I'll leave this issue open as still integration for `explain_prediction` is missing.

**Comment by Stuart-D-King** [comment]
@ivanprado @AshwinB-hat 
Sorry, but I can't seem to get this to work. Here's some code:

`expl = eli5.explain_weights(cb_model)`
`print(eli5.format_as_text(expl))`
>>Error: estimator <catboost.core.CatBoostClassifier object at 0x13b579400> is not supported

Can someone tell me what I'm doing wrong? Thanks.

**Comment by lopuhin** [comment]
@Stuart-D-King this code is in master but not released yet, it should work if you install eli5 from github (something like ``pip install https://github.com/TeamHG-Memex/eli5/archive/master.zip``).

**Comment by Stuart-D-King** [comment]
@lopuhin Ahhh, thanks so much!

**Comment by lopuhin** [comment]
``eli5.explain_weights`` support for catboost released as part of 0.9.0, thanks @AshwinB-hat for implementation and @ivanprado for review!
